### ISSUE TITLE
Hovering over icon in systray shows weird menu

### ISSUE THREAD

**Issue by Darkmaster006** [clementine-player/Clementine#7074]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Linux Mint 20.2 Uma
- Clementine version: 1.4 rc1

### Expected behaviour / actual behaviour

![image](https://user-images.githubusercontent.com/21136515/130125289-54734bd9-1865-4af1-9d4d-44a93b416372.png)
Basically, this is what appears when I hover over the icon, which I guess isn't supposed to appear like that. Any ideas? Thank you.

**Comment by jbroadus** [comment]
Which desktop are you using?

**Comment by Darkmaster006** [comment]
> Which desktop are you using?

Cinnamon.

Here's inxi -Fxz output for System:
System:
  Kernel: 5.4.0-74-generic x86_64 bits: 64 compiler: gcc v: 9.3.0 
  Desktop: Cinnamon 5.0.5 Distro: Linux Mint 20.2 Uma 
  base: Ubuntu 20.04 focal

**Comment by fitojb** [comment]
Quasi-duplicate of #6793
